File: Assets/Scripts/messages/PointCloudMessage.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class PointCloudMessage
{
    public List<string> point_json;

    public PointCloudMessage(List<string> point_json)
    {
        this.point_json = point_json;
    }

    public static PointCloudMessage FromJson(string json)
    {
        return JsonUtility.FromJson<PointCloudMessage>(json);
    }

    public string ToJson()
    {
        return JsonUtility.ToJson(this);
    }
}

File: python/messages/point_cloud_message.py
import json
import numpy as np

class PointCloudMessage:
    def __init__(self, point_json):
        self.point_json = point_json

    @staticmethod
    def from_json(json_string):
        point_cloud_message_dict = json.loads(json_string)
        return PointCloudMessage(point_cloud_message_dict["point_json"])

    def to_json(self):
        return json.dumps(self.__dict__)